File-Formats/PDF/Convert-HTML-To-PDF/troubleshooting.md

```diff
@@ -424,6 +424,56 @@ blinkConverterSettings.CommandLineArguments.Add("--disable-setuid-sandbox");
 </tr>
 </table>
 
+## Mac Machine Error: "Exception has been thrown by the target of an invocation"
```

Comment:
OCR not working on Mac: Exception has been thrown by the target of an invocation

Reply:
Updated the title